streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/UserService.java

```diff
@@ -120,6 +120,8 @@ public interface UserService extends IService<User> {
 
     void setLatestTeam(Long teamId, Long userId);
 
+    void clearDeletedTeamId(Long userId, Long teamId);
```

Comment:
how about `unbindTeam(Long userId, Long teamId)` ?

Reply:
Hi @wolfboys , thanks for your review. I addressed all comments~